recipes/mimeo/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
```

Comment:
I think this needs a change: https://bioconda.github.io/linting.html#setup-py-install-args

Reply:
Fixed in latest commit, thanks.